tests/testthat/tests_imports.R

```diff
@@ -188,7 +188,7 @@ test_that("importWaterML2 external test", {
   xml <- getWebServiceData(url)
   measurementTS2 <- xml2::xml_find_all(xml, "//wml2:MeasurementTimeseries")
   exData <- importWaterML2(measurementTS2)
-  expect_is(exData$value, 'numeric')
+  expect_is(exData$value, c('numeric', 'logical'))
```

Comment:
Presumably, this test will go back to all numeric's as soon as the ice melts. Would it be a more stable test if it was:
```
expect_true(class(exData$value) %in% c('numeric', 'logical'))
```
I'm actually still a bit confused why it would come out as logical...even if they are all NA's...I'd think we'd want it numeric NA's...found the problem line I think above.